plox.py

```diff
@@ -90,7 +91,9 @@ def run(self, source: str):
             return
 
         try:
-            self.interpreter.interpret(statements)
+            last_value_produced = self.interpreter.interpret(statements)
+            if self.in_repl and last_value_produced is not None:
```

Comment:
No lo testee, pero este `is not None`, prevendría que se imprima el resultado si termina siendo `Nil`? Es una decision adrede?

Reply:
Buenísimo! Tiene sentido